packages/terra-functional-testing/tests/jest/webpack-server/webpack-server.test.js

```diff
@@ -89,6 +103,270 @@ describe('Webpack Server', () => {
+      const compiler = {
+        watch: jest.fn(),
+        hooks: {
+          done: {
+            tap: jest.fn((arg1, callback) => {
+              callback(mockStats);
+            }),
+          },
+          failed: {
+            tap: () => {},
+          },
+        },
+        options: {
+          devServer: {},
+        },
+      };
+      webpack.mockReturnValue(compiler);
+
+      const server = new WebpackServer();
+
+      return expect(server.start()).resolves.toBeUndefined();
+    });
+
+    it('rejects if WebpackDevServer.listen errors', async () => {
+      jest.spyOn(WebpackServer, 'config').mockImplementation(() => 'config');
+      const compiler = {
+        watch: jest.fn(),
+        hooks: {
+          done: {
+            tap: () => {},
+          },
+          failed: {
+            tap: () => {},
+          },
+        },
+        options: {
+          devServer: {},
+        },
+      };
+      webpack.mockReturnValue(compiler);
+      const mockMethod = jest.fn((arg1, arg2, callback) => {
+        callback(true);
+      });
+      WebpackDevServer.mockImplementation(() => ({ listen: mockMethod }));
+
+      const server = new WebpackServer();
+      server.gridStatusUrl = '0.0.0.0:8080';
+
+      return expect(server.start()).rejects.toBe(true);
+    });
+
+    it('throws if grid is not ready', async () => {
+      const chunk = {
+        value: {
+          ready: false,
+        },
+      };
+      const onfn = jest.fn()
+        .mockImplementationOnce((event, cb) => cb(JSON.stringify(chunk)))
+        .mockImplementationOnce((event, cb) => cb());
+      const response = {
+        statusCode: 200,
+        on: onfn,
+        complete: true,
+      };
+      jest.spyOn(WebpackServer, 'config').mockImplementation(() => 'config');
+      jest.spyOn(http, 'get').mockImplementation((arg1, callback) => {
+        callback(response);
+      });
+      const mockStats = {
+        hasErrors: jest.fn().mockImplementationOnce(() => false),
+      };
+      const compiler = {
+        watch: jest.fn(),
+        hooks: {
+          done: {
+            tap: jest.fn((arg1, callback) => {
+              callback(mockStats);
+            }),
+          },
+          failed: {
+            tap: () => {},
+          },
+        },
+        options: {
+          devServer: {},
+        },
+      };
+      webpack.mockReturnValue(compiler);
+
+      const server = new WebpackServer();
+      server.gridStatusUrl = '1.1.1.1:8080';
+      await expect(server.start()).rejects.toBeUndefined();
+      expect(mockLoggerInstance.error.mock.calls[0][0]).toEqual(`${server.gridStatusUrl} failed to return a ready response. Check to ensure the selenium grid is stable`);
+      // expect.assertions(2);
+      // return expect(server.start()).rejects.toBe(false);
+      // return server.start().catch(e => expect(e.message).toEqual(`${server.gridStatusUrl} failed to return a ready response. Check to ensure the selenium grid is stable`));
```

Comment:
Are these needed?

Reply:
They are not, removed here https://github.com/cerner/terra-dev-site/pull/367